lib/task-data/schemas/dell-wsman-get-config-catalog.json

```diff
@@ -0,0 +1,81 @@
+{
+    "copyright": "Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "serverIP": {
+            "description": "Dell server iDARC IP address",
+            "type": "string"
+        },
+        "serverUsername": {
+            "description": "Dell server iDARC username",
+            "type": "string"
+        },
+        "serverPassword": {
+            "description": "Dell server iDARC password",
+            "type": "string"
+        },
+        "shareAddress": {
+            "description": "NFS/CIFS server address which holds firmware repository",
+            "type": "string"
+        },
+        "shareUsername": {
+            "description": "NFS/CIFS server usename",
+            "type": "string"
+        },
+        "sharePassword": {
+            "description": "NFS/CIFS server password",
+            "type": "string"
+        },
+        "shareName": {
+            "description": "Share path of the firmware repository folder on share server",
+            "type": "string"
+        },
+        "shareType": {
+            "description": "Specify whehter use NFS or CIFS as share server to hold firmware repository",
+            "type": "integer",
+            "enum": [0, 2]
+        },
+        "shutdownType": {
+            "description": "Specify the type of system shutdown: Graceful(0), Forced(1), default value:0",
```

Comment:
The description tells the default `shutdownType` is zero, but I don't see the default value in task definition: https://github.com/RackHD/on-tasks/pull/564/files#diff-a4208a0c727e4c4e91866bbc80e01457R20